workspaces/tech-insights/.changeset/sixty-chicken-wave.md

```diff
@@ -0,0 +1,9 @@
+---
+'@backstage-community/plugin-tech-insights-backend': patch
+'@backstage-community/plugin-tech-insights-node': patch
+---
+
+Inetgrate notifications plugin into the tech-insights.
+
+If the notifications are enabled, the user will be able to access the
```

Comment:
Can we also add documentation for this - as in how do adopters of the plugin actually use it.